src/main/java/it/gov/pagopa/payment/service/QRCodePaymentServiceImpl.java

```diff
@@ -51,4 +63,12 @@ public TransactionResponse confirmPayment(String trxId, String merchantId) {
     return qrCodeConfirmationService.confirmPayment(trxId, merchantId);
   }
 
+  @Override
+  public SyncTrxStatusDTO getStatusTransaction(String transactionId, String merchantId, String acquirerId) {
```

Comment:
Let's move the business logic in a separated service as for the other APIs
In this class we will just call the external service

Reply:
maybe just this time, only because there are very low code's rows, we could let it here :) 

When we will implement the CIEPaymentService, this entire class will be renamed into "BasePaymentService" (making a new QRCodePaymentService which will extend it), this way also the CIE flows will ineherit this logic